src/mame/mame.lst

```diff
@@ -13782,6 +13782,7 @@ bm37th                          // GCB07 (c) 2002 (Japan JCA)
 bm3core                         // GCA05 (c) 2000 (Japan)
 bm3final                        // GCC01 (c) 2003 (Japan JCA)
 kbm                             // GQ974 (c) 2000 (Japan)
+kbh                             // GU974 (c) 2000 (Korea)
```

Comment:
Please keep this in alphabetical order.

Reply:
it's a clone, should the clone really be listed first?

Reply:
The `.lst` and `.flt` files are just sorted for the most part to make them more manageable.